### PR TITLE
licenses: add mplus, yasashisa-gothic: init at 0-unstable-2014-03-13

### PR DESCRIPTION
**Pull request by h7x4**
## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/yasashisa-gothic/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchzip, ... }:
+stdenvNoCC.mkDerivation rec {
+  pname = "yasashisa-gothic";
+  version = "unstable-2023-10-06";
```
Comment (pbsds) [inline pkgs/data/fonts/yasashisa-gothic/default.nix:4]: The newest date in the zip is 2014-03-13

**Review by pbsds — CHANGES_REQUESTED** [review]
The [spdx identifier](https://spdx.org/licenses/mplus.html
) is correct, just a minor nit, then LGTM

- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package build on x86_64-linux
- [x] executables tested on x86_64-linux
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.platforms` is set
- [x] `meta.maintainers` is set
- [x] build time only dependencies are declared in `nativeBuildInputs`
- [x] source is fetched using the appropriate function
- [x] the list of `phases` is not overridden
- [x] when a phase (like `installPhase`) is overridden it starts with `runHook preInstall` and ends with `runHook postInstall`.
- [x] patches have a comment describing either the upstream URL or a reason why the patch wasn't upstreamed
- [x] patches that are remotely available are fetched rather than vendored

---

Result of `nixpkgs-review pr 259421` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>nix-init</li>
    <li>yasashisa-gothic</li>
  </ul>
</details>

**Comment by pbsds** [comment]
Darwin doesn't seem to like the zip

**Comment by h7x4** [comment]
Drafting this for now. The original zip was created on a windows machine using [`CP932`](https://en.wikipedia.org/wiki/Code_page_932_(Microsoft_Windows)), but the conversion is so broken that it needs to be reconverted back to UTF-8 before it's unpacked. This needs to be done file by file, so the conversion magic would need to be embedded into the tool. We do have such a tool, [`runzip`](https://github.com/NixOS/nixpkgs/blob/nixos-23.11/pkgs/tools/archivers/runzip/default.nix), but it appears to be slightly broken.

**Comment by pbsds** [comment]
Result of `nixpkgs-review pr 259421` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>nix-init</li>
    <li>tests.nixpkgs-check-by-name</li>
    <li>yasashisa-gothic</li>
  </ul>
</details>


LGTM

**Review by pbsds — APPROVED** [review]

**Pull request merged**